CONTRIBUTING.md

```diff
@@ -54,19 +54,198 @@ development.
    under `images`, but only step 7 if you changed things only under `jupyterhub`
 
 
-## Releasing a new version of the Helm Chart
-
-In order to release a new version of the Helm Chart, make sure to perform each
-of the following steps:
-
-1. Generate a release document. This should follow the structure of previous
-   entries in the [CHANGELOG](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/CHANGELOG.md).
-2. Double-check that there aren't any un-documented breaking changes.
-3. If applicable, include a section in the [CHANGELOG](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/CHANGELOG.md)
-   that contains upgrade instructions (usually only applicable for major changes to the code).
-4. Generate a list of contributors since the latest release. Use the script
-  in `tools/contributors.py` to list all contributions (anyone who made a
-  commit or a comment)    since the latest release. Update the dates in that
-  script, run it, and paste the output into the changelog. For an example,
-  see [the v0.5 list of contributors](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/v0.5/CHANGELOG.md#contributors).
-5. ...
+## Releasing a new version of the helm chart
+
+The following steps can be followed to release a new version of the Helm Chart.
+Presently, we expect a release approximately every 5-7 weeks.
+
+
+### Create an issue for the new release
+
+Use this issue to coordinate efforts and keep track of progress. You can
+copy / paste the raw Markdown from the following list, which will be covered
+in more detail below.
+
+```
+Title: Release {{release-name}}
+Content:
+
+This issue will be used to coordinate the next release of the helm
+chart, {{release-name}}. Instructions for creating the release can be found in
+[CONTRIBUTING.md](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-helm-chart).
+Below is the checklist for this release.
+
```

Comment:
I would like to an explicit step 0: Code, tests, and documentation to support a release are stable.

Reply:
I'm firm on wanting this explicitly stated. This should be the goal even if it takes a release or two to get there.

Reply:
Definitely agree w/ you on this point 👍 my question only pertained to the other comment